mkosi/__init__.py

```diff
@@ -6743,49 +6743,47 @@ def find_qemu_binary() -> str:
 
 
 def find_qemu_firmware() -> Tuple[str, bool]:
-    # UEFI firmware blobs are found in a variety of locations,
-    # depending on distribution and package.
-    FIRMWARE_LOCATIONS = []
-
-    if platform.machine() == "x86_64":
-        FIRMWARE_LOCATIONS.append("/usr/share/ovmf/x64/OVMF_CODE.secboot.fd")
-    elif platform.machine() == "i386":
-        FIRMWARE_LOCATIONS.append("/usr/share/edk2/ovmf-ia32/OVMF_CODE.secboot.fd")
-
-    FIRMWARE_LOCATIONS.append("/usr/share/edk2/ovmf/OVMF_CODE.secboot.fd")
-    FIRMWARE_LOCATIONS.append("/usr/share/qemu/OVMF_CODE.secboot.fd")
-    FIRMWARE_LOCATIONS.append("/usr/share/ovmf/OVMF.secboot.fd")
+    FIRMWARE_LOCATIONS = [
+        # UEFI firmware blobs are found in a variety of locations,
+        # depending on distribution and package.
+        {
+            "x86_64": "/usr/share/ovmf/x64/OVMF_CODE.secboot.fd",
+            "i386": "/usr/share/edk2/ovmf-ia32/OVMF_CODE.secboot.fd",
+        }.get(platform.machine()),
```

Comment:
what happens on platforms other than x8664/i386 with this expression? Doesn't this resolve to None then? Is that desirable?